WordPressAuthenticatorTests/Services/LoginFacadeTests.m

```diff
@@ -17,6 +18,10 @@
 __block id mockLoginFacadeDelegate;
 __block LoginFields *loginFields;
 
+beforeAll(^{
+    [WordpressAuthenticatorProvider initializeWordPressAuthenticator];
```

Comment:
I was getting a crash in these tests since `WordPressAuthenticator` wasn't initialized.